scripts/tests/test_suites.sh

```diff
@@ -107,16 +110,13 @@ for j in "${iter_array[@]}"; do
         # tee expeditiously; otherwise it will buffer things up and we
         # will never see the string we want.
 
-        # Clear out our temp files so we don't accidentally do a stale
-        # read from them before we write to them.
-        rm -rf /tmp/"$application"-log
-        touch /tmp/"$application"-log
+        touch /tmp/test_suites_app_logs/"$application"-log
```

Comment:
No, you really do need to delete this log.  We grep for things in it, and if there's data in there from a previous iteration or previous test from this iteration, it will fail.

The other option is to add the iteration number (`j`) and test name (`i`) to the file name, so each one has a separate log file and out grep bits work right.

Reply:
Ok, makes sense. I didn't realize that the same application is run multiple times for different test cases.